compiler/src/dotty/tools/dotc/transform/init/Objects.scala

```diff
@@ -1485,12 +1485,12 @@ class Objects(using Context @constructorOnly):
       if isWildcardStarArgList(pats) then
         if pats.size == 1 then
           // call .toSeq
-          val toSeqDenot = scrutineeType.member(nme.toSeq).suchThat(_.info.isParameterless)
+          val toSeqDenot = getMemberMethod(scrutineeType, nme.toSeq, toSeqType(elemType)) //scrutineeType.member(nme.toSeq).suchThat(_.info.isParameterless)
```

Comment:
Is this a refactoring or a bug fix? We can remove the commented code.

Reply:
This is just a refactor. I forgot to remove it after I tested the code to see if it still worked. 

Side note, I'll make sure rebase next time :sweat_smile:

Reply:
No worries, it's easy to rebase and squash --- you can force push to the PR branch.